lib/wallet/src/Cardano/Wallet.hs

```diff
@@ -1711,7 +1712,27 @@ increaseZeroAdaOutputs era pp = WriteTx.modifyLedgerBody $
             then WriteTx.computeMinimumCoinForTxOut era pp out
             else c
 
+
+-- | Given the input and output 'Cardano.Tx' from 'balanceTransaction', this
+-- function will identify the change outputs which were added.
+-- changeFromBalancing
+--     :: Cardano.Tx era
+--     -> Cardano.Tx era
+--     -> [Cardano.TxOut Cardano.CtxTx era]
+-- changeFromBalancing
+
+data BalancingResultInfo s = BalancingResultInfo
+    { addedChange :: [Address]
```

Comment:
I don't think `addedChange :: [Address]` is useful. Perhaps `addedChange :: [WriteTx.TxOut era]` is useful... (in some API endpoints change outputs are returned separately from normal outs) ... otherwise I'll remove it though.

Reply:
I dropped it and the record in favour of just `s' :: s`